src/extension/types/tournamentData.ts

```diff
@@ -0,0 +1,152 @@
+export interface Rules {
```

Comment:
This is specific to Battlefy - I'd rather the file name be `battlefyTournamentData`